### PR TITLE
Show Last sync time

### LINKED ISSUE #1: Add the last-synced time of a link to the link detail page.
Currently, it's not possible to see the last-sync time of a Backstroke link in the dashboard. This is available within the [`link-detail` component](https://github.com/backstrokeapp/dashboard/tree/master/src/components/link-detail) as `this.props.initialLinkState.lastSyncedAt`. Ideally, the last-synced time should be placed below the switch and resync buttons in the UI, something like the below, though if you have a better and more creative solution, that would be even better!

screen shot 2017-10-08 at 11 24 18 am

### PR DESCRIPTION
**Pull request by recrsn**
Shows last synced time.

Fixes #1 

screenshot_20171008_222135

### CODE REVIEW

File: src/components/link-detail/styles.css [hunk 1]
```diff
@@ -302,3 +302,8 @@
   display: inline-block;
   color: #de3535;
 }
+
+.link-detail-last-sync-time {
```
Comment (1egoman) [inline src/components/link-detail/styles.css:306]: Would you be able to make the font a bit bigger, maybe 18 or 24px? Also, could you use flexbox to right align the text? I suspect that will help visually link the text to the "resync" button.

File: src/components/link-detail/index.js [hunk 2]
```diff
@@ -197,6 +199,11 @@ export class LinkDetail extends React.Component {
           >{link.lastWebhookSync && link.lastWebhookSync.status === 'TRIGGERED' ? 'Waiting...' : 'Resync'}</Button>
         </div>
 
+        { link.lastSyncedAt ?
```
Comment (1egoman) [inline src/components/link-detail/index.js]: Can you format `{ link.lastSyncedAt ? <div className="link-detail-row">` on one line?

Also, why is the ternary required? I think `link.lastSyncedAt` should always exist, if I remember right.

File: src/components/link-detail/index.js [hunk 3]
```diff
@@ -197,6 +199,11 @@ export class LinkDetail extends React.Component {
...
+        <div className="link-detail-row">
+          <span className="link-detail-last-sync-time">Synced <TimeAgo date={ link.lastSyncedAt } /></span>
+        </div>
```
Comment (1egoman) [inline src/components/link-detail/index.js]: Can you format `</div : null }` on one line?

**Review by 1egoman — CHANGES_REQUESTED** [review]
I mentioned few small things that would be great to have adjusted prior to this being merged, but overall, you're most of the way there!  I really appreciate when other people take their time to help out.

File: src/components/link-detail/index.js (hunk 2 above)
Comment (recrsn) [reply src/components/link-detail/index.js]: I think the same form is used for creating the link. But if its always available it should not be required.
Though I haven't gone through the server and worker code yet!

Comment (1egoman) [reply src/components/link-detail/index.js]: Oh, that's true. When creating the link I'm not sure that the last synced time will exist. A ternary probably should be here.

File: src/components/link-detail/styles.css (hunk 1 above)
Comment (recrsn) [reply src/components/link-detail/styles.css:306]: 18px now, aligned right

**Comment by recrsn** [comment]
screenshot-2017-10-9 backstroke

**Comment by 1egoman** [comment]
Thanks!

**Pull request merged**